contrib/download-frozen-image-v2.sh

```diff
@@ -63,10 +63,10 @@ fetch_blob() {
 			-o "$targetFile" \
 			-D-
 	)"
-	curlHeaders="$(echo "$curlHeaders" | tr -d '\r')"
```

Comment:
Is moving this line important?  The new `grep` should match better with proper `\n` than with additional `\r`, shouldn't it?